### PR TITLE
JetBrains: Rename "recipes" to "commands" in docs and UI

### PR DESCRIPTION
**Pull request by vdavid**
- It only needed a single rename on the UI
- Updated docs

## Test plan

- I've read through the docs to make sure they make sense
- Searched all files in JetBrains to make sure we have no other visible uses of the term "recipe"

### CODE REVIEW

File: client/jetbrains/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For example, you can ask Cody:
 
 ### 🤖 Ask Cody about anything in your codebase
 
-Cody understands your entire codebase — not just your open files. Ask questions, insert code, and use the built-in recipes such as "Summarize recent code changes" and "Improve variable names".
```
Comment (vdavid) [inline client/jetbrains/README.md:21]: I sneaked this in, but it should really be a part of my other PR that uses the agent for recipes.

File: client/jetbrains/README.md [hunk 2]
```diff
@@ -28,20 +28,17 @@ Cody can provide real-time code autocompletions as you're typing. As you start c
...
+## 🍳 Built-in commands
 
-Select the recipes tab or right-click on a selection of code and choose one of the `Ask Cody > ...` recipes, such as:
+Select the commands tab or right-click on a selection of code and choose one of the `Ask Cody > ...` commands, such as:
 
 - Explain code
 - Generate unit test
 - Generate docstring
 - Improve variable names
-- Translate to different language
```
Comment (vdavid) [inline client/jetbrains/README.md:39]: Also should be a part of the other PR

File: client/jetbrains/src/main/java/com/sourcegraph/cody/CodyToolWindowContent.java [hunk 3]
```diff
@@ -95,7 +95,7 @@ public CodyToolWindowContent(@NotNull Project project) {
     tabbedPane.insertTab("Chat", null, contentPanel, null, CHAT_TAB_INDEX);
     @NotNull JPanel recipesPanel = new JPanel(new GridLayout(0, 1));
     recipesPanel.setLayout(new BoxLayout(recipesPanel, BoxLayout.Y_AXIS));
-    tabbedPane.insertTab("Recipes", null, recipesPanel, null, RECIPES_TAB_INDEX);
```
Comment (vdavid) [inline client/jetbrains/src/main/java/com/sourcegraph/cody/CodyToolWindowContent.java:98]: Funny how this was the only use of the term

**Review by olafurpg — COMMENTED** [review]
Delegating this review to @taylorsperry

**Review by taylorsperry — APPROVED** [review]

**Pull request merged**